### ISSUE TITLE
SmartCity-Accessibility Server Side Network Tests #347 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#463]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.31 KB...]
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 7, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 24.666 sec <<< FAILURE! - in smartcity.accessibility.services.NavigationServiceTest
responseLatLngArrayIsNotEmpty(smartcity.accessibility.services.NavigationServiceTest)  Time elapsed: 1.336 sec  <<< FAILURE!
java.lang.AssertionError: Range for response status value 400 expected:<SUCCESSFUL> but was:<CLIENT_ERROR>
	at smartcity.accessibility.services.NavigationServiceTest.responseLatLngArrayIsNotEmpty(NavigationServiceTest.java:100)

Running smartcity.accessibility.navigation.NavigationTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 8.738 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.754 sec - in smartcity.accessibility.database.ParseDatabaseTest
Running smartcity.accessibility.search.NearbyPlacesTest
the length is : 10
lat is : 31.9031344 lng is : 34.9968192
lat is : 31.9075046 lng is : 34.9979821
lat is : 31.9075329 lng is : 34.997949
lat is : 31.9092985 lng is : 35.00175489999999
lat is : 31.908945 lng is : 34.996069
lat is : 31.9076557 lng is : 34.9981921
lat is : 31.9006521 lng is : 35.002661
lat is : 31.9005432 lng is : 35.0025467
lat is : 31.9121792 lng is : 35.00472180000001
lat is : 31.9014627 lng is : 35.0013844
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.12 sec - in smartcity.accessibility.search.NearbyPlacesTest

Results :

Failed tests: 
  NavigationServiceTest.responseLatLngArrayIsNotEmpty:100 Range for response status value 400 expected:<SUCCESSFUL> but was:<CLIENT_ERROR>

Tests run: 20, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:19 min
[INFO] Finished at: 2017-06-28T16:40:35+00:00
[INFO] Final Memory: 32M/182M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] d32cf35b0d4849f2f27cacc9d01e8ddcbfb1fd19 - set default value to rating #459

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/347/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
